x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/tabular_page.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import {} from './constants';
+
+import { EuiFlexGroup, EuiButton, EuiFlexItem, EuiPageTemplate } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { InferenceAPIConfigResponse } from '@kbn/ml-trained-models-utils';
+
+import {
+  InferenceEndpointUI,
+  INFERENCE_ENDPOINTS_TABLE_PER_PAGE_VALUES,
+  SortFieldInferenceEndpoint,
+  SortOrder,
+} from './types';
+
+import { useAllInferenceEndpointsState } from './use_all_inference_endpoints_state';
+
+import { InferenceEndpointsProvider } from '../../providers/inference_endpoints_provider';
+import { EndpointsTable } from './endpoints_table';
+import { TABLE_COLUMNS } from './table_columns';
+
+interface TabularPageProps {
+  addEndpointLabel: string;
+  breadcrumbs: string[];
+  inferenceEndpoints: InferenceAPIConfigResponse[];
+  setIsInferenceFlyoutVisible: (value: boolean) => void;
+}
+
+const getSortField = (field: string): SortFieldInferenceEndpoint => {
+  if (field in SortFieldInferenceEndpoint) {
+    return SortFieldInferenceEndpoint[field as keyof typeof SortFieldInferenceEndpoint];
+  }
+  return SortFieldInferenceEndpoint.endpoint;
+};
+
+export const TabularPage: React.FC<TabularPageProps> = ({
+  addEndpointLabel,
+  breadcrumbs,
```

Comment:
This is unused, assuming since this component is rendered in serverless only and that's handled automatically. If so we should remove this prop as not needed.

Reply:
Updated